sound/soc/soc-topology.c

```diff
@@ -623,7 +623,13 @@ static int soc_tplg_kcontrol_bind_io(struct snd_soc_tplg_ctl_hdr *hdr,
 	}
 
 	/* vendor specific handlers found ? */
-	if (k->put && k->get && k->info)
+	/* RW controls need to have both get/put IO handlers */
```

Comment:
@RDharageswari these checks are for the io_ops and not for the ext_ops right? Do we really need to check these here?

Reply:
Removed the handling here